variants/rakwireless_rak11300/common.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
```

Comment:
This is the same as https://github.com/earlephilhower/arduino-pico/blob/master/variants/generic/common.h. New variants should not copy this file into their variant board folder, but just reference it with

https://github.com/earlephilhower/arduino-pico/blob/87f6646e959646ef96f6d7ceb71bdd9ac973395d/variants/adafruit_feather/pins_arduino.h#L44

in their `pins_arduino.h` file.

Reply:
Thanks, done :)